### PR TITLE
Change fzaninotto/faker to fakerphp/faker since the former is deprecated

### PR DESCRIPTION
**Pull request by rihardsgrislis**
Hey,
Backstory: https://twitter.com/taylorotwell/status/1321091021342650377
Can this be changed now? Or there are other steps to take before this can be done?

### CODE REVIEW

**Review by owenvoke — APPROVED** [review]
Personally I don't see any issue with this, although not sure as there is a conflict between `fakerphp/faker` and `fzaninotto/faker`.

I think it should be fine though as anyone still using `fzaninotto/faker` would download `v0.3.0` of this, and anyone else would download `v0.3.1`. 🤷🏻

Also, it looks like this has been changed in Laravel core as well. @nunomaduro, do you think this would be too early?

**Comment by nunomaduro** [comment]
@GrahamCampbell

**Comment by GrahamCampbell** [comment]
Leave it as `^1.9.1`. The 1.9.1 release is very special because the new package 1.9.1 release is marked as replacing the old 1.9.1 package, so if you have a (indirect) dependency on the old package anywhere, you can still use version 1.9.1 of the new package.

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "license": "MIT",
     "require": {
         "php": "^7.3 || ^8.0",
-        "fzaninotto/faker": "^1.9.1",
+        "fakerphp/faker": "^1.10",
```
Comment (owenvoke) [inline composer.json]: As per Graham's comment:
```suggestion
        "fakerphp/faker": "^1.9.1",
```

**Comment by owenvoke** [comment]
@nunomaduro, is it ok to merge and tag this as v0.3.1? 👍🏻

**Comment by nunomaduro** [comment]
If think is ok, is ok. 👍

**Pull request merged**